### PR TITLE
#2085 control-click from start page should open in new window

### PR DESCRIPTION
**Pull request by dlukyanyuk**

### CODE REVIEW

**Comment by samccone** [comment]
Can we just use an anchor tag here? Feels like we are reinventing the wheel

**Comment by scottdraves** [comment]
anchor tag sounds good, is that possible dmitry?

**Comment by scottdraves** [comment]
our style guide says no tabs, and indentation of JS is two spaces.
https://github.com/twosigma/beaker-notebook/wiki/Style-Guide

File: core/src/main/web/app/styles/dashboard.scss [hunk 1]
```diff
@@ -81,4 +81,9 @@
       color: $green;
     }
   }
-}
+
+  .notebook-link {
+    display: block;
+    color: #333333;
+  }
+}
```
Comment (scottdraves) [inline core/src/main/web/app/styles/dashboard.scss]: end files with newline

File: core/src/main/web/app/controlpanel/table.jst.html [hunk 2]
```diff
@@ -28,4 +28,4 @@
       {{session.openedDate | date:'medium'}}
     </div>
   </li>
-</ul>
+</ul>
```
Comment (scottdraves) [inline core/src/main/web/app/controlpanel/table.jst.html]: end files with newline

**Comment by scottdraves** [comment]
:+1:

**Pull request merged**